WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -1591,3 +1591,17 @@ extension WooAnalyticsEvent {
         WooAnalyticsEvent(statName: .universalLinkFailed, properties: [Key.url.rawValue: url.absoluteString])
     }
 }
+
+// MARK: - Jetpack connection
+//
+extension WooAnalyticsEvent {
+    enum LoginJetpackConnection {
+        enum Key: String {
+            case selfHosted = "is_selfhosted_site"
```

Comment:
```suggestion
            case selfHosted = "is_self_hosted_site"
```

Nit: Should we add an underscore to follow the naming from [the authenticator library](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/trunk/WordPressAuthenticator/Analytics/AuthenticatorAnalyticsTracker.swift#L41)?

Reply:
Since this has already been used on Android, I'll keep it as-is 😅